shoop/core/order_creator/_modifier.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.db.transaction import atomic
+
+from shoop.core.models import Order
+
+from ._creator import OrderProcessor
+
+
+class OrderModifier(OrderProcessor):
+
+    @atomic
+    def update_order_from_source(self, order_source, order):
+        data = self.get_source_base_data(order_source)
+        Order.objects.filter(pk=order.pk).update(**data)
+
+        order = Order.objects.get(pk=order.pk)
+
+        for line in order.lines.all():
+            line.delete()
```

Comment:
This will make all rows linked to OrderLine vanish. Or if there is any model that links to OrderLine with on_delete=PROTECT FK then this will fail. I guess that is OK though, but should be documented so that developers using Shoop will know to not link to lines but to the order.